Command/AnnotatedCommandTrait.php

```diff
@@ -22,7 +34,53 @@ private function setAnnotatedTasks()
 
             foreach ($annotations as $annotation) {
                 if ($annotation instanceof Schedule) {
-                    array_push($this->tasks, $annotation->toArray());
+                    $annotationTask = $annotation->toArray();
+
+                    $task = [];
+                    foreach (AttributeInterface::ATTRIBUTES as $attribute) {
+                        if (AttributeInterface::ATTRIBUTE_ID == $attribute) {
+                            // Generate Id.
+                            $id = HashHelper::generateIdFromProps(
+                                ArrayHelper::only($annotationTask, HashHelper::GENERATED_PROPS)
+                            );
+
+                            $task[$attribute] = $id;
+
+                            continue;
+                        }
+
+                        if (AttributeInterface::ATTRIBUTE_STATUS == $attribute) {
+                            if (!isset($annotationTask[$attribute])) {
```

Comment:
if ve elseif blokları tek bir or'lu if bloğu içinde yazılabilir, ikisinde de işlem aynı, hatta !isset de null eşitliği olmadan tek başına yeterli olabilir:

```suggestion
                            if (!isset($annotationTask[$attribute]) || null == $annotationTask[$attribute]) {
                                $task[$attribute] = StatusInterface::STATUS_ACTIVE;
                            }
```

ya da 

```suggestion
                            if (!isset($annotationTask[$attribute])) {
                                $task[$attribute] = StatusInterface::STATUS_ACTIVE;
                            }
```